internal/commands/exec/command.go

```diff
@@ -229,3 +266,18 @@ Usage: consul-api-gateway exec [options]
 	Handles service registration, certificate rotation, and spawning envoy.
 `
 }
+
+func (c *Command) parseDiscoveryAddresses() (addresses string, port int, err error) {
```

Comment:
the conflation of `CONSUL_HTTP_ADDR` really only happens in the server entrypoint since we've always distinguished address v. port for the purposes of our deployments -- so this logic shouldn't be necessary here.